### DISCUSSION TITLE
Feature Request - ability to customize shortcut for cycling through tabs

### DISCUSSION THREAD

**Discussion by YaYaTurre** [zen-browser/desktop#2026] [Ideas]
### Captchas

- [X] I have read the instructions.
- [X] I have searched existing issues and avoided creating duplicates.
- [X] I am not filing an enhancement request.

### What happened?

I would like to ability to customize the key combination for cycling through tabs in a window. Ctrl + Tab is not a comfortable combination on a Mac.

### Reproducible?

- [X] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.1-a.8 (aarch64)

### What platform are you seeing the problem on?

macOS - aarch64

**Comment by linear** [comment]
<p><a href="https://linear.app/zen-browser/issue/ZEN-1945/feature-request-ability-to-customize-shortcut-for-cycling-through-tabs">ZEN-1945 Feature Request - ability to customize shortcut for cycling through tabs</a></p>

**Comment by 4rtemis-4rrow** [comment]
yes please

I like vimium's shortcuts, but they only work when the page finishes loading, sometimes I want to switch tabs before that happens, being able to map shift+j and shift+k for that would be awesome

**Comment by Thenlie** [comment]
Plus one.

In addition to being able to remap cycling tabs, I would like a custom shortcut for "Next Tab" and "Previous Tab"

**Comment by YaYaTurre** [comment]
I have started using the existing keyboard shortcuts for cycling through tabs in a window: Opt + Cmd + left/right arrow. (on Mac)

This is what I was really looking to do when I created this feature request.

**Reply by Thenlie** [reply]
Ah yeah that is what I was looking for as well. Just would like the ability to re-map this now :)

**Comment by vodkya** [comment]
This was previously a feature in Zen (next tab and previous tab keybinds), and I think it should be counted as a regression. It's been rather annoying not being able to rebind this since updating, as Ctrl+Tab is an ergonomic nightmare.

**Comment by Spurkus** [comment]
It seems like they are adding it back in this pull request https://github.com/zen-browser/desktop/pull/3599
